src/misc.ts

```diff
@@ -293,7 +293,16 @@ export function getMutationStatus<TData extends Record<string, SaleorMutationRes
 ): ConfirmButtonTransitionState {
   const errors = getMutationErrors(opts);
```

Comment:
question: could you check what `getMutationErrors` does? I think it actually checks `opts.errors`, so change here is unnecessary. If it doesn't then we probably should add error handling there